.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: false
+language: python
+python:
+  - 2.7
```

Comment:
This was failing when I tried using Python 3.4. I guess it was choking on the whole `print` vs `print()` fiasco.